frontends/ol-components/src/components/LearningResourceCard/LearningResourceCard.tsx

```diff
@@ -243,11 +231,7 @@ const LearningResourceCard: React.FC<LearningResourceCardProps> = ({
   return (
     <StyledCard href={href} className={className} size={size}>
       <Card.Image
-        src={
-          resource.image?.url
-            ? getEmbedlyUrl(resource, size, isMedia)
-            : DEFAULT_RESOURCE_IMG
-        }
+        src={resource.image?.url ? resource.image?.url : DEFAULT_RESOURCE_IMG}
```

Comment:
@jonkafton Did you mean to use both embedly and NextJS Image component here? I've switched it to just NextJS for now.

Reply:
I added a fallback in `getEmbedlyUrl` in the Storybook branch to just use the image URL if the env var is not set: 

```
  if (!process.env.NEXT_PUBLIC_EMBEDLY_KEY!) {
    return resource.image!.url!
  }
```

Writing this I realize that where we use the nextjs/image component we'd be pulling an Embedly image via the Next.js image optimizer, so yeh - we don't want to do that!